spec/definitions.yaml

```diff
@@ -840,10 +840,10 @@ definitions:
         type: object
         properties:
           min:
-            description: Minimum number of nodes in the pool
+            description: Minimum number of nodes in the pool, must be greater than 0, less than `max`.
```

Comment:
"less than max" is not exactly right. min can be equal to max. This is the setting that also disables autoscaling.